### PR TITLE
ui/alerts-custom-json

### PR DESCRIPTION
**Pull request by imnotashrimp**
# What changed

- Added details for sending custom fields with json.
- Updated output screen shot
- Updated query screen shot (might take a few hours to clear the cache)

----

## Pages to review

- https://deploy-preview-465--logz-docs.netlify.app/user-guide/alerts/configure-an-alert.html#output-format

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

To be completed by the docs team upon merge:

no tasks

### CODE REVIEW

File: _source/user-guide/alerts/configure-alerts.md [hunk 1]
```diff
@@ -151,19 +151,20 @@ You can search triggered alert logs at any time.
 
 If you chose recipients, you can also set the output format.
 
-![Output table](https://dytvr9ot2sszz.cloudfront.net/logz-docs/alerts/output-table.png)
+![Output table](https://dytvr9ot2sszz.cloudfront.net/logz-docs/alerts/output-json-custom-fields.png)
 
-You can choose to send **All fields** in JSON format,
-or **Custom fields** as a table.
+Choose whether to send the data as **JSON** or in a **Table**.
```
Comment (idohalevi) [inline _source/user-guide/alerts/configure-alerts.md:156]: My English is not as good as yours but still, I'll try... you use or to give two options, is it correct that one of them is JSON format and the other is a location(in a table). If the label is `Output format` isn't it suppose to say `...as a JSON or a Table`?

Comment (idohalevi) [reply _source/user-guide/alerts/configure-alerts.md:156]: In addition, let's make sure they understand it's a sample data no?

File: _source/user-guide/alerts/configure-alerts.md [hunk 2]
```diff
@@ -151,19 +151,20 @@ You can search triggered alert logs at any time.
...
 (minimum, maximum, average, or sum),
-the output will use only grouped or aggregated fields.
-To change these fields, you'll need to change your **Group by** or **Trigger if...** settings.
+the output will include only the grouped or aggregated fields.
+To change these fields, you'll need to first change your **Group by** or **Trigger if...** settings.
 {:.info-box.note}
 
-If you choose a table,
-click **<i class="li li-plus"></i> Add a field** to add a column to the table.
+If you choose a table or JSON with custom fields,
```
Comment (idohalevi) [inline _source/user-guide/alerts/configure-alerts.md:166]: This proves my point about `a JSON or a Table` -- here you refer to both as formats.

File: _source/user-guide/alerts/configure-alerts.md [hunk 3]
```diff
@@ -151,19 +151,20 @@ You can search triggered alert logs at any time.
...
+click **<i class="li li-plus"></i> Add a field** to add the field to your output.
 You can optionally sort one field or filter the output with a regular expression.
```
Comment (idohalevi) [inline _source/user-guide/alerts/configure-alerts.md:168]: Do we want to add something about the fact that if users want to send `all fields` we support json only?

File: _source/user-guide/alerts/configure-alerts.md (hunk 1 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md:156]: "a JSON", not really - "a JSON object", maybe. I'll keep that part as is - but definitely yes on reminding the user it's a sample. That'll be in my next push to this PR.

File: _source/user-guide/alerts/configure-alerts.md (hunk 3 above)
Comment (imnotashrimp) [reply _source/user-guide/alerts/configure-alerts.md]: Good point. Revised a paragraph further up to clarify this. In the next push.

**Pull request merged**